src/styles/app-modal/app-dialog/AppDialog.types.ts

```diff
@@ -22,4 +28,5 @@ export type AppDialogProps = {
   cancelButton?: boolean | AppDialogButtonConfig;
   shouldDisableOverlayClick?: boolean;
   isFullscreen?: boolean;
+  tooltipOverflowText?: AppDialogTooltipConfig;
```

Comment:
```suggestion
  titleTooltipConfig?: AppDialogTooltipConfig;
```

And place it under title

Reply:
Already fixed